app/controllers/api/v1/accounts/following_accounts_controller.rb

```diff
@@ -22,7 +22,7 @@ def load_accounts
 
     scope = default_accounts
     scope = scope.not_excluded_by_account(current_account) unless current_account.nil? || current_account.id == @account.id
-    scope.merge(paginated_follows).to_a
+    scope.merge(paginated_follows).includes(:user).to_a
```

Comment:
Wouldn't it make more sense to put that in the existing `includes` in `default_accounts`?